plugins/inputs/cisco_telemetry_mdt/sample.conf

```diff
@@ -50,3 +53,5 @@
   ## GRPC minimum timeout between successive pings, decreasing this value may 
   ## help if this plugin is closing connections with ENHANCE_YOUR_CALM (too_many_pings).
   # keepalive_minimum_time = "5m"
+
+
```

Comment:
Please only add one newline if at all.

Reply:
Solved in [8123f7e](https://github.com/influxdata/telegraf/pull/12345/commits/8123f7e46f4f13847d1020c12fcb398ea1f13312) respectively with update of the docs in [36355d8](https://github.com/influxdata/telegraf/pull/12345/commits/36355d8c9cb9442ffbf4aeb21efde42d82578a64)